aspnetcore/release-notes/aspnetcore-7.0.md

```diff
@@ -187,6 +187,14 @@ One place where these improvements can be noticed is in gRPC, a popular RPC fram
 
 ![Entity diagram](https://user-images.githubusercontent.com/219224/177910504-e93579b4-02e4-4079-8a8c-d9d24857aabf.png)
 
+### Kestrel performance improvements on high core machines
+
+Kestrel uses <xref:System.Collections.Concurrent.ConcurrentQueue%601> for many purposes. One purpose is scheduling I/O operations in Kestrel's default Socket transport. Partitioning the `ConcurrentQueue` based on the associated socket reduces contention and increases throughput on machines with many CPU cores.
+
+There are even higher core machines, such as the 80 core ARM64 Ampere Altra VMs that were recently made available on Azure. Profiling with these machines in .NET 6 showed a lot of contention in one of Kestrel's other `ConcurrentQueue` instances, the `PinnedMemoryPool` that Kestrel uses to cache byte buffers.
```

Comment:
```suggestion
Profiling on high core machines on .NET 6 showed significant contention in one of Kestrel's other `ConcurrentQueue` instances, the `PinnedMemoryPool` that Kestrel uses to cache byte buffers.
```

Reply:
Can you add link to build https://review.docs.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-7.0?view=aspnetcore-7.0&branch=pr-en-us-26745#kestrel-performance-improvements-on-high-core-machines
so reviewer doesn't need to look that up?